lisa/schema.py

```diff
@@ -834,6 +861,7 @@ class Runbook:
     parent: Optional[List[Parent]] = field(default=None)
     extension: Optional[List[Union[str, Extension]]] = field(default=None)
     variable: Optional[List[Variable]] = field(default=None)
+    transformer: Optional[List[Transformer]] = field(default=None)
```

Comment:
Shall we stop using singular form for things that are clearly lists? This is very confusing for users. CC @WenyiQ

Reply:
Agree. I also prefer to use plural form for lists personally.

Reply:
It's a good discussion. Firstly, it's cost to update existing ones, and keeping consistence is clearer to me. Secondly, to use plural form or not, it depends on thinking from different angles, if it's a noun, it should indicate it. If it's a concept, or the namespace of some information, it may not need. anyway user needs to learn to know the keyword. If some of them are singular, and some of them are plural, it makes another confusing. Users may be confused on it needs an "s" or not. For example, the platform is defined as a list for future extensibility, but for now, single platform is required, and it's easy to understand.

Reply:
User confusion is one having to refer to docs (or either worse, .py schemas) to see if something is a list or not, while it could for sure be a name hint. I know of the costs to fix the current state, but one carefully crafted patch should do it. As for the extensibility for the future, we just change it again (we can ditch the list from platforms, for now, everywhere).